product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx

```diff
@@ -90,11 +75,15 @@ The following table describes the parameters that need to be entered in the EDB
 ![*Subscription progress](images/subscription-process.png "image_tooltip")
 
 4. Select **Configure account now** to configure EDB Cloud to use your Azure subscription and your Azure AD Application.
-5. Fill in the form using the values of parameters described in [Required Parameters](#Required-Parameters).
-   1. Azure AD App ID
-   2. Azure AD App Secret
-   3. Azure Subscription ID
-   4. Your EDB Cloud Organization Name
+5. Fill in the following parameters in the form: form using the values of parameters you noted when [creating your Azure AD Application](#create-azure-ad-application) and your EDB Cloud Origanization Name found on the overview page of a subscription.
```

Comment:
in the section above this where is says to enter SaaS subscription name - that's in the subscription details section not the project details section - and we'll want to rever to that field name for Your EDB Cloud Organization I think